app/src/components/v-upload.vue

```
@@ -86,9 +121,7 @@ function useUpload() {
 		};
 
 		try {
-			if (!validFiles(files)) {
-				throw new Error('An error has occurred while uploading the files.');
-			}
+			if (!validFiles(files)) return;
```

Does this now silently ignore?

---

The error is not handled in `validFiles` function

---

Ah you're doing the notifications in validFiles, gotcha